internal/compiler-bridge/src/main/scala/xsbt/ExtractUsedNames.scala

```diff
@@ -49,29 +52,39 @@ import Compat._
 class ExtractUsedNames[GlobalType <: CallbackGlobal](val global: GlobalType) extends Compat with ClassName with GlobalHelpers {
   import global._
 
-  def extract(unit: CompilationUnit): Iterable[(String, Iterable[String])] = {
+  def extract(unit: CompilationUnit): HashMap[String, ArrayList[String]] = {
```

Comment:
Fixed, parially, I can replace the HashMap with Map, but replacing ArrayList with Iterable gives me this:
```
Error:(87, 5) type mismatch;
 found   : java.util.HashMap[String,java.util.ArrayList[String]]
 required: java.util.Map[String,Iterable[String]]
Note: java.util.ArrayList[String] <: Iterable[String], but Java-defined trait Map is invariant in type V.
You may wish to investigate a wildcard type such as `_ <: Iterable[String]`. (SLS 3.2.10)
    result
```